docs/framework/docker/aspnetmvc.md

```diff
@@ -0,0 +1,241 @@
+on the Docker site. 
+
+The application you'll run in a container is a simple website that
+answers questions randomly. This application
+is a basic MVC application with no authentication support, or database
+storage. That lets you focus on moving the web tier to a container. Later
+topics show how to move and manage persistent storage in containerized
+applications.
+
+Moving your application involves these steps:
+
+1. [Creating a publish task to build the assets for an image.](#publish-script)
+2. [Building a Docker image that will run your application.](#build-the-image)
+3. [Starting a Docker container that runs your image.](#start-a-container)
+4. [Verifying the application using your browser.](#verify-in-the-browser)
+
+The finished application is located in the
+[dotnet/core-docs repository on GitHub](https://github.com/dotnet/core-docs/tree/master/samples/framework/docker/MVCRandomAnswerGenerator).
+
+## Prerequisites
+
+Your development machine must be running Windows 10 Anniversary Update, or newe,
+or, Windows Server 2016, or newer.
+
+Before starting, you need to install [Docker for Windows](https://docs.docker.com/docker-for-windows/).
+You need to install version 1.12 Beta 26, or newer for Windows container support.
+After installing and starting Docker, you'll need to right-click on the
+tray icon and select **Switch to Windows containers...** in order to run
+Docker images based on Windows. This command takes a few seconds to
+execute:
+
+![Windows Container][windows-container]
+
+
+## Publish script
+
+You can use the Visual Studio Publish command to create a publish profile
+for your application. You'll copy all the published assets to your Docker
+image later in this tutorial. The publish command will put all the publish
+assets in one directory tree that you will copy to your target image.
+Right-click on your solution in Visual Studio, and select "Publish".
+Click the "Custom" profile button, and then select "File System" as the
+method (see image). Choose a new directory in your solution folder. (The
+downloaded sample uses "containerImage" as the folder name).
+
+![Publish Connection][publish-connection]
+
+Next, open the "File Publish Options" section of the "Settings" tab. Select
+"Precompile during publishing". This optimization means that you be
+compiling views in the Docker container, you are copying the precompiled
+views.
+
+![Publish Settings][publish-settings]
+
+Click Publish, and Visual Studio will copy all the needed assets to the
+destination folder. 
+
+## Build the image
+
+To run your application, you'll build an image based on the `microsft/aspnet`
+image located on [Docker Hub](https://hub.docker.com/r/microsoft/aspnet/).
+
+The base image, `microsoft/aspnet` is a Windows Server image. In addition
+to the Windows Server Core, it has IIS enabled. It also has ASP.NET 4.6.2
+installed and enabled. When you run this image in a container, it will start
+IIS and any installed websites will be active.
```

Comment:
> The base image, microsoft/aspnet is a Windows Server image.

I think a comma is missing after microsoft/aspnet? 

>In addition to the Windows Server Core, it has IIS enabled. It also has ASP.NET 4.6.2 installed and enabled. 

In addition to the Windows Server Core, it has IIS and .NET Framework 4.6.2 installed and enabled. 

>When you run this image in a container, it will start IIS and any installed websites will be active.

Is this automatic? If so, would be good to maybe say "... it will automatically start IIS..."